src/populate-error-message.ts

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-var has = require('lodash.has');
-var is = require('is');
+import * as has from 'lodash.has';
+import * as is from 'is';
```

Comment:
It'd be good if imports are sorted by the package names. I think `gts fix` does that, doesn't it?